runtime/astar/src/lib.rs

```diff
@@ -2185,29 +2206,7 @@ impl_runtime_apis! {
     }
 }
 
-struct CheckInherents;
```

Comment:
Maybe a dumb question - why is it ok to remove this?

If I understand correctly, in th eplace of this struct, `DummyCheckInherents` will be used instead which does nothing.

Reply:
From migration guide https://wiki.polkadot.network/docs/maintain-guides-async-backing
> If your runtime/src/lib.rs provides a CheckInherents type to register_validate_block, remove it. FixedVelocityConsensusHook makes it unnecessary. The following example shows how register_validate_block should look after removing CheckInherents.